libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -14,18 +14,18 @@ ifdef::apm-server[]
 endif::[]
 
 To monitor {beatname_uc}, make sure monitoring is enabled on your {es} cluster,
-then configure the method used to collect {beatname_uc} metrics. You
-ifndef::serverless[]
-can use one of following methods:
-endif::[]
-ifdef::serverless[]
-can use the following method:
-endif::[]
+then configure the method used to collect {beatname_uc} metrics. You can use one
+of following methods:
 
-* <<monitoring-internal-collection,Internal collection>>
 ifndef::serverless[]
-* <<monitoring-metricbeat-collection, {metricbeat} collection>>
+* <<monitoring-metricbeat-collection, {metricbeat} collection>> - 
+{metricbeat} sends monitoring data directly to your monitoring cluster.
 endif::[]
+* <<monitoring-internal-collection,Internal collection>> - Internal
```

Comment:
I think this terminology might be confusing, since we use "Internal collection" for other products to mean "internal collection sending data to the production cluster" (what you're calling "legacy collection" below).

E.g., in Logstash monitoring docs: https://www.elastic.co/guide/en/logstash/current/configuring-logstash.html.